subsys/bluetooth/host/mesh/net.c

```diff
@@ -68,6 +68,7 @@ static u16_t msg_cache_next;
 
 /* Singleton network context (the implementation only supports one) */
 struct bt_mesh_net bt_mesh = {
+	.local_queue = _K_FIFO_INITIALIZER(bt_mesh.local_queue),
```

Comment:
Why the underscore prefix?

Reply:
You should really ask the core kernel maintainers, but to my understanding this got added to those kernel APIs that in the long run will only be usable from other kernel code, i.e. user space code will not be able to use them.

Reply:
These initializer types require special handling.
For the purposes of user-mode permission tracking, you have to call _k_object_init() on objects initialized in this way, for those types of objects that we expose to userspace via system calls.
For this case it's moot because 1) this is kernel only code and 2) fifo/lifo/queues are not exposed as system calls due to the way they are designed